documentation/how-to/upgrade-server.md

```diff
@@ -0,0 +1,45 @@
+# Upgrade Temporal Server
+
+Excerpt from
+[Temporal's documentation](https://docs.temporal.io/self-hosted-guide/upgrade-server):
+
+> Temporal Server should be upgraded sequentially. That is, if you're on version
+> (v1.n.x), your next upgrade should be to (v1.n+1.x) or the closest available
+> subsequent version. This sequence should be repeated until your desired
+> version is reached.
+>
+> 1. Upgrade Database Schema: Before initiating the Temporal Server upgrade, use
+>    one of the recommended upgrade tools to update your database schema. This
+>    ensures it is aligned with the version of Temporal Server you aim to
+>    upgrade to.
+> 2. Upgrade Temporal Server: Once the database schema is updated, proceed to
+>    upgrade the Temporal Server deployment to the next sequential version.
+>
+> Also, be aware that each upgrade requires the History Service to load all
+> Shards and update the Shard metadata, so allow approximately 10 minutes on
+> each version for these processes to complete before upgrading to the next
+> version.
+
+The Charmed Temporal K8s charms facilitate server upgrades in the following way:
+
+1. The Charmed Temporal Admin charm should be updated to the next revision that
```

Comment:
For users that are not very familiar with Charms it could be worth adding a link to [juju revision](https://juju.is/docs/sdk/revision) documentation. Or clarifying that a revision isn't the same as Temporal version as the two are sort of mixed here.

Reply:
Good point, thanks!